ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/ConfigCommitter.scala

```diff
@@ -98,25 +98,41 @@ private[kvutils] case class ConfigCommitter(
             buildRejectionLogEntry(
               ctx,
               result.submission,
-              _.setInvalidConfiguration(
-                DamlConfigurationRejectionEntry.InvalidConfiguration.newBuilder
-                  .setError(err)))),
+              _.setInvalidConfiguration(Invalid.newBuilder
+                .setDetails(err)))),
         config =>
           if (config.generation != (1 + result.currentConfig._2.generation))
             StepStop(
               buildRejectionLogEntry(
                 ctx,
                 result.submission,
-                _.setGenerationMismatch(
-                  DamlConfigurationRejectionEntry.GenerationMismatch.newBuilder
-                    .setExpectedGeneration(1 + result.currentConfig._2.generation))
+                _.setGenerationMismatch(GenerationMismatch.newBuilder
+                  .setExpectedGeneration(1 + result.currentConfig._2.generation))
               )
             )
           else
             StepContinue(result)
       )
   }
 
+  private val deduplicateSubmission: Step = (ctx, result) => {
+    val submissionKey = configDedupKey(ctx.getParticipantId, result.submission.getSubmissionId)
+    if (ctx.get(submissionKey).isEmpty) {
+      logger.info(s"Accepting submissionId ${result.submission.getSubmissionId}")
```

Comment:
shouldn't this be trace log?

Reply:
Removed altogether. It was for debugging